src/app/blog/page.tsx

```
@@ -1,51 +1,54 @@
+interface ArtigosListaData {
+	listaArtigos: {
+		titulo: string;
+		buttonLabel: string;
+		buttonLink: string;
+	}[];
+}
+
+const HomeBlogPage = async () => {
+	const artigosData = (await getData(POSTS)) as { blogArtigos: Artigo[] };
+	const artigos = artigosData
+		? artigosData.blogArtigos.map((artigo: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)